packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -181,9 +163,13 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
       return null;
     }
 
-    const itemKey: string = this.state.selectedKey;
+    const itemKey: string | undefined = this._selectedKey();
+    if (!itemKey) {
+      return null;
+    }
+
     const index = this._keyToIndexMapping[itemKey];
-    const { selectedTabId } = this.state;
+    const selectedTabId = this._selectedTabId();
```

Comment:
Since this calculation is only done once, it doesn't really need to be in a method (especially since the method currently redoes the selectedKey calculation). Also, it could be simplified to:
```
const selectedTabId = this._keyToTabIds[itemKey];
```
(if `itemKey` is undefined, `this._keyToTabIds[itemKey]` should also be undefined and won't error)